### DISCUSSION TITLE
Delaying deploys until GitHub action or GitLab pipeline finishes

### DISCUSSION THREAD

**Discussion by kvnwolf** [vercel/community#76] [Idea]
How can I add a Vercel check that waits for the checks on that specific commit before creating the deployment?

https://vercel.com/guides/how-can-i-use-gitlab-pipelines-with-vercel

**Comment by barelyhuman** [comment]
I don't see a "recommended way" to do it , but it's a simple github integration and I guess writing a wrapping integration over it would be the only alternative right now. 

I'd need more context though, 
why do you want to block parallel deployments? It's a preview url , unless you only use the `main` branch for everything

**Comment by Pieparker** [comment]
3-off requests for delayed deploys in this now-locked discussion: https://github.com/vercel/community/discussions/97
